### PR TITLE
[MOD-2369] Android: Add "addPolylines" method (Parity with iOS)

### PR DESCRIPTION
**Pull request by jvandijk**
**JIRA**: https://jira.appcelerator.org/browse/MOD-2369

Although the documentation states, or does not explicitly state the support on Android for the `addPolylines` method, it doesn't exist. This PR introduces the addPolylines method to create parity with the iOS implementation

### CODE REVIEW

**Comment by hansemannn** [comment]
@jvandijk Please bump the Android manifest to the next feature version (should be 3.4.0).

**Comment by jvandijk** [comment]
@hansemannn any idea if this could make the 7.0.0 release?

**Comment by hansemannn** [comment]
@jvandijk Now that the build passes we can integrate it asap. Let us review it this week!

File: android/src/ti/map/ViewProxy.java [hunk 1]
```diff
@@ -780,7 +789,50 @@ public void handleAddPolyline(Object polyline) {
...
+			}
+			setProperty(MapModule.PROPERTY_POLYLINES, polylinesList.toArray());
+		} else {
+			setProperty(MapModule.PROPERTY_POLYLINES, lines);
+		}
+
+		TiUIView view = peekView();
+		if (view instanceof TiUIMapView) {
+			if (TiApplication.isUIThread()) {
+				handleAddAnnotations(lines);
```
Comment (hansemannn) [inline android/src/ti/map/ViewProxy.java]: Shouldn't this be `handleAddPolylines`?

Comment (jvandijk) [reply android/src/ti/map/ViewProxy.java]: You are absolutely right.... copy & paste error!

**Comment by jvandijk** [comment]
Bump!

**Comment by hansemannn** [comment]
@jvandijk I just released 4.1.0 via https://github.com/appcelerator-modules/ti.map/releases/tag/android-4.1.0. Can you confirm it includes your change and works properly? We then target it for SDK 7.1.0.

**Pull request merged**